cpp/models/abm/location.h

```diff
@@ -396,12 +281,9 @@ class Location
     }
 
 private:
-    std::mutex m_mut; ///< Mutex to protect the list of persons from concurrent modification.
-    LocationId m_id; ///< Id of the Location including type and index.
-    bool m_capacity_adapted_transmission_risk; /**< If true considers the LocationCapacity for the computation of the 
-    transmission risk.*/
+    LocationType m_type; ///< Type of the Location.
+    LocationId m_id; ///< Id of the Location. Set by the World owning it.
```

Comment:
```suggestion
    LocationId m_id; ///< LocationId of the Location. Set by the World owning it.
```

Reply:
I changed it to "Unique identifier for the Location in the World owning it" instead?

Reply:
Same as above.